views/components/feedbackBanner/feedbackBanner.njk

```diff
@@ -0,0 +1,9 @@
+<div class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    <div class="feedback-banner" data-qa="feedback-banner">
+      <a href="https://eu.surveymonkey.com/r/GYB8Y9Q?source={{ hostname + currentUrlPath }}" class="govuk-link govuk-link--inverse" target="_blank" rel="noopener noreferrer">
+        Give feedback on this service
+      </a>
+    </div>
+  </div>
+</div>
```

Comment:
the little red thing  is saying you are missing a new line at t he end of the file

NBD but technically files without new lines at the end aren't valid files

bit annoying we don't have linting rules to check this